impl/src/main/java/org/jboss/cdi/tck/tests/full/interceptors/tests/contract/method/MethodLevelInterceptorTest.java

```diff
@@ -49,7 +51,7 @@ public void testInterceptorCanBeAppliedToMoreThanOneMethod() {
         assertEquals(FishInterceptor.getInstanceCount(), 1);
     }
 
-    @Test
+    // Use of @ExcludeClassInterceptors not supported in Lite
```

Comment:
But this class is now in the `full` package, and annotated `@Test(groups = CDI_FULL)`, so this should remain unchanged?